week6/src/CreateTask.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState,useEffect } from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import { Link} from 'react-router-dom';
+import List from './List';
+
+
+function CreateTask({modal,toggle}) {
+    const storedTask = JSON.parse(localStorage.getItem('items')) || []
+    const storedDescription = JSON.parse(localStorage.getItem('descript')) || []
+  const [tasks, setNewitem]= useState("");
+  const [items, setItems] = useState(storedTask);
+  const [descriptions, setNewDescription]= useState("");
+  const [descript, setDescription] = useState(storedDescription);
+  useEffect(() => {
+    localStorage.setItem('items', JSON.stringify(items));
+    
+  }, [items]);
+  useEffect(() => {
+    localStorage.setItem('descript', JSON.stringify(descript));
+    
+  }, [descript]);
+  function add(){
+    if (!tasks){
+      alert("please input task!");
+      return;
+    }
+    if (!descriptions){
+        alert("please input description!");
+        return;
+      }
+    const taskinput = /^(?=.*[a-z])/;
+    if (!taskinput.test(tasks)){
+      alert('must contain at least one letter');
```

Comment:
The alert says 'must contain at least ....' but does not make it clear that it is referred to the task name? 
Same thing as description below